content/source/docs/enterprise/system-overview/architecture.html.md

```diff
@@ -5,24 +5,202 @@ page_title: "Architecture Summary - System Overview - Terraform Enterprise"
 
 # Terraform Enterprise Architecture
 
-This page describes aspects of the architecture of Terraform Enterprise.
+Terraform Enterprise (TFE) is powered by the same set of services behind HashiCorp’s public SaaS application Terraform Cloud.  The sections below present the functional layers of the architecture, how they interact and details pertaining to the corresponding docker containers involved.  References to Docker containers in this document are those you will see when running a `docker ps` on a TFE application server while the application is up.
```

Comment:
We generally avoid abbreviating the product name anywhere on our website where we can avoid it. TFE should be spelled out as Terraform Enterprise throughout out website documentation.